### PR TITLE
Update test spec

### PR DESCRIPTION
**Pull request by eldadfux**
Following this PR: https://github.com/appwrite/appwrite/pull/955, I have updated the Swagger test file we use to generate tests, the new spec will allow us to generate test SDKs that have new method that will allow us to call the new endpoints added to appwrite/appwrite. For the new endpoint to work we need to merge both PR, and deploy a new version of appwrite.io

### CODE REVIEW

File: tests/resources/spec.json [hunk 1]
```diff
@@ -593,29 +603,86 @@
...
+        "security": [{ "Project": [] }]
+      }
+    },
     "/mock/tests/general/empty": {
       "get": {
         "summary": "Mock a post request for SDK tests",
         "operationId": "generalEmpty",
         "consumes": ["application/json"],
-        "produces": ["application/json"],
+        "produces": [],
```
Comment (lohanidamodar) [inline tests/resources/spec.json:667]: Not sure why this was changed everywhere? What's the significance?

File: tests/resources/spec.json [hunk 2]
```diff
@@ -593,29 +603,86 @@
...
+    "/mock/tests/general/400-error": {
+      "get": {
+        "summary": "Mock a an 400 failed request",
+        "operationId": "generalError400",
+        "consumes": ["application/json"],
+        "produces": [],
+        "tags": ["general"],
+        "description": "",
+        "responses": {
+          "500": { "description": "File", "schema": { "type": "file" } }
```
Comment (lohanidamodar) [inline tests/resources/spec.json:615]: Response, Description and schema seems wrong, this should be 400 not 500 and not file

File: tests/resources/spec.json [hunk 3]
```diff
@@ -593,29 +603,86 @@
...
+    "/mock/tests/general/500-error": {
+      "get": {
+        "summary": "Mock a an 500 failed request",
+        "operationId": "generalError500",
+        "consumes": ["application/json"],
+        "produces": [],
+        "tags": ["general"],
+        "description": "",
+        "responses": {
+          "500": { "description": "File", "schema": { "type": "file" } }
```
Comment (lohanidamodar) [inline tests/resources/spec.json:643]: Description and schema seems wrong

**Review by lohanidamodar — CHANGES_REQUESTED** [review]

File: tests/resources/spec.json (hunk 2 above)
Comment (eldadfux) [reply tests/resources/spec.json:615]: 👍🏻 I'll fix that

File: tests/resources/spec.json (hunk 3 above)
Comment (eldadfux) [reply tests/resources/spec.json:643]: I'll fix that

**Comment by eldadfux** [comment]
Fixed the spec file, this will also help us to test the future response formats :)

**Review by lohanidamodar — APPROVED** [review]

File: tests/resources/spec.json (hunk 1 above)
Comment (eldadfux) [reply tests/resources/spec.json:703]: This is used mainly in the docs, because of a bug it was missing the previous time I generated the spec

**Pull request merged**